### PR TITLE
add roadupdate aug2024

### PR DESCRIPTION
**Pull request by gregbunce**
add road update post and update trails and pathways dataset date

### CODE REVIEW

**Comment by gregbunce** [comment]
looks good to me. let me know if missed anything, else please approve and i'll merge it later. thanks!

File: src/content/blog/2024-08-09-utah-sgid-statewide-roads-data-layer-updates.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+#### County Updates
+
+New roads were added and road names and address ranges were updated for the following counties:
+
+- **Davis County:** Obtained roads data on 07/15/2024. Previous update was on 06/14/2024.
+- **Salt Lake County:** Obtained roads data on 07/15/2024. Previous update was on 07/19/2024.
+- **Utah County:** Obtained roads data on 07/15/2024. Previous update was on 07/19/2024.
+- **Washington County:** Obtained roads data on 07/30/2024. Previous update was on 07/19/2024.
+- **Weber County:** Obtained roads data on 07/22/2024. Previous update was on 07/19/2024.
```
Comment (stdavis) [inline src/content/blog/2024-08-09-utah-sgid-statewide-roads-data-layer-updates.md:26]: Some of these dates look a little suspicious. For example, SLC data obtained on 7/15/24 but previous update was 7/19/2024? Would you like double-check them?

**Review by steveoh — APPROVED** [review]

File: src/content/blog/2024-08-09-utah-sgid-statewide-roads-data-layer-updates.md (hunk 1 above)
Comment (steveoh) [reply src/content/blog/2024-08-09-utah-sgid-statewide-roads-data-layer-updates.md:26]: I thought the same but it seems reasonable that we got new data after we did the last update.

**Comment by gregbunce** [comment]
yeah, those are the dates that i show. @MichaelAGRC do you want to verify those dates? thanks!

**Review by stdavis — APPROVED** [review]

**Pull request merged**